server/accounts.go

```diff
@@ -416,6 +417,12 @@ func (a *Account) AddServiceExportWithResponse(subject string, respType ServiceR
 	return nil
 }
 
+// AddServiceExportWithresponse will configure the account with the defined
+// export and response type.
+func (a *Account) AddServiceExportWithResponse(subject string, respType ServiceRespType, accounts []*Account) error {
+	return a.AddServiceExportWithResponseAndLatency(subject, respType, serviceLatency{}, accounts)
+}
+
```

Comment:
Not too sure about this part. I created another function so I could pass `serviceLatency` to `exportAuth`. Is this okay? Or should I just add a param to the original function? Seems like we would break the API if we add a param, but not sure if we allow people to use those functions on their own.

Reply:
There is a separate function that can add latency tracking to an export. Let's use that. You can find it in the original tests.

Reply:
Okay, checked out some of the tests. Did you mean I could use `TrackServiceExportWithSampling`?

I tried something like this in the `parseAccounts` function.
```go
if err := service.acc.AddServiceExportWithResponse(service.sub, service.rt, accounts); err != nil {
	msg := fmt.Sprintf("Error adding service export %q: %v", service.sub, err)
	*errors = append(*errors, &configErr{tk, msg})
	continue
}

if service.lat != nil {
	err := service.acc.TrackServiceExportWithSampling(
		service.sub, service.lat.subject, int(service.lat.sampling),
	)
	if err != nil {
		msg := fmt.Sprintf("Error adding latency metrics for %s: %v", service.sub, err)
		*errors = append(*errors, &configErr{tk, msg})
		continue
	}
}
```

But, I keep hitting the error case.
```
Error adding latency metrics for nats.add: system account not setup
```

I noticed in the tests we use this.
```
if err := s.SetSystemAccount(globalAccountName); err != nil {
	t.Fatalf("Error setting system account: %v", err)
}
```

But, since I'm using the config file, am I going to need something like this, instead?
```
operator: /Users/synadia/.nsc/nats/memory/memory.jwt
resolver: MEMORY
resolver_preload: {
	ACSU3Q6LTLBVLGAQUONAGXJHVNWGSKKAUA7IY5TB4Z7PLEKSR5O6JTGR: eyJ0eXAiOiJqd3QiLCJhbGciOiJlZDI1NTE5In0.eyJqdGkiOiJPRFhJSVI2Wlg1Q1AzMlFJTFczWFBENEtTSDYzUFNNSEZHUkpaT05DR1RLVVBISlRLQ0JBIiwiaWF0IjoxNTU2NjU1Njk0LCJpc3MiOiJPRFdaSjJLQVBGNzZXT1dNUENKRjZCWTRRSVBMVFVJWTRKSUJMVTRLM1lERzNHSElXQlZXQkhVWiIsIm5hbWUiOiJBIiwic3ViIjoiQUNTVTNRNkxUTEJWTEdBUVVPTkFHWEpIVk5XR1NLS0FVQTdJWTVUQjRaN1BMRUtTUjVPNkpUR1IiLCJ0eXBlIjoiYWNjb3VudCIsIm5hdHMiOnsibGltaXRzIjp7InN1YnMiOi0xLCJjb25uIjotMSwibGVhZiI6LTEsImltcG9ydHMiOi0xLCJleHBvcnRzIjotMSwiZGF0YSI6LTEsInBheWxvYWQiOi0xLCJ3aWxkY2FyZHMiOnRydWV9fX0._WW5C1triCh8a4jhyBxEZZP8RJ17pINS8qLzz-01o6zbz1uZfTOJGvwSTS6Yv2_849B9iUXSd-8kp1iMXHdoBA
}

accounts: { ... }
```
¿?

Reply:
You do not need full on operator mode, but yes you need a system account.